tests/CSharp/CSharp.Tests.cs

```diff
@@ -854,4 +854,14 @@ public void TestVoidPtrReturningIndexer()
             Assert.That(*((int*) indexproperty[n]), Is.EqualTo(21));
         }
     }
+
+    [Test]
+    public void TestFuncWithTypedefedFuncPtrAsParam()
+    {
+        using (var hasFuncWithTypedefedFuncPtrAsParam = new HasFuncWithTypedefedFuncPtrAsParam())
+        {
+            CSharp.Delegates.Func_int_IntPtr_CSharp_Bar___Internal function = (System.IntPtr a, CSharp.Bar.__Internal b) => 5;
```

Comment:
Replace the fully qualified name with a using. Remove the parameter types of the lambda.